app/assets/javascripts/application.js

```diff
@@ -558,7 +574,9 @@ function disableButtonToggle(item, explicit) {
   $(item).blur();
 }
 
-function icon(name) {
+function icon_text(name, inner_text, icon_class) {
   "use strict";
-  return '<span class="glyphicon glyphicon-' + name + '" />';
+  var icon = '<i class="' + icon_class + " " + icon_class + "-" + name + '"/>'
```

Comment:
I'd say use `<span>`, even if font-awesome for some reason chose to use `<i>` in their examples as `<span>` is the correct one according to the HTML specs.